docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.8"
+version: "3.7"
```

Comment:
docker on my machine did not like 3.8... no clue why, not really looking to dig into that problem unless there is reason to.. It liked 3.7 though.